lib/sequence.js

```diff
@@ -9,99 +8,67 @@ function Sequence(tests, bail) {
 		throw new TypeError('Class constructor Sequence cannot be invoked without \'new\'');
 	}
 
-	this.tests = tests;
-	this.bail = bail;
+	this.results = [];
+	this.passed = true;
+	this.reason = null;
+	this.tests = tests || [];
```

Comment:
Seems unnecessary to default to an empty array, just let it throw `this.tests is undefined` if it is misused.